pkg/etl/config/etl_profile.h

```diff
@@ -0,0 +1,53 @@
+/*
+* Copyright (C) 2022 Jens Wetterich <jens@wetterich-net.de>
+*
+* This file is subject to the terms and conditions of the GNU Lesser
+* General Public License v2.1. See the file LICENSE in the top level
+* directory for more details.
+*/
+
+/**
+* @ingroup     pkg_etl
+* @brief
+* @{
+*
+* @file
+* @brief       Common settings for etl
+* @details     This activates the usage of the STL if available and/or the usage of workarounds for missing NAN macros.
```

Comment:
```suggestion
* @details     This activates the usage of the STL if available and/or the usage of
               workarounds for missing NAN macros.
```

The CI is very upset when lines are longer than 100 chars :/